ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-v0.19.0__4.6.yaml

```diff
@@ -0,0 +1,259 @@
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/hellohttp2
+  to: knative-serving-test-hellohttp2
+- dockerfile_path: openshift/ci-operator/knative-test-images/failing/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/failing
+  to: knative-serving-test-failing
+- dockerfile_path: openshift/ci-operator/knative-test-images/timeout/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/timeout
+  to: knative-serving-test-timeout
+- dockerfile_path: openshift/ci-operator/knative-test-images/pizzaplanetv2/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/pizzaplanetv2
+  to: knative-serving-test-pizzaplanetv2
+- dockerfile_path: openshift/ci-operator/knative-test-images/hellovolume/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/hellovolume
+  to: knative-serving-test-hellovolume
+- dockerfile_path: openshift/ci-operator/knative-test-images/varlog/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/varlog
+  to: knative-serving-test-varlog
+- dockerfile_path: openshift/ci-operator/knative-test-images/wsserver/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/wsserver
+  to: knative-serving-test-wsserver
+- dockerfile_path: openshift/ci-operator/knative-test-images/runtime/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/runtime
+  to: knative-serving-test-runtime
+- dockerfile_path: openshift/ci-operator/knative-test-images/singlethreaded/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/singlethreaded
+  to: knative-serving-test-singlethreaded
+- dockerfile_path: openshift/ci-operator/knative-test-images/httpproxy/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/httpproxy
+  to: knative-serving-test-httpproxy
+- dockerfile_path: openshift/ci-operator/knative-test-images/flaky/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/flaky
+  to: knative-serving-test-flaky
+promotion:
+  name: release-v0.19.0
+  namespace: openshift
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.1"
+  namespace: ocp
+test_binary_build_commands: make test-install
+tests:
+- as: e2e-aws
+  commands: make test-e2e
+  openshift_installer_src:
```

Comment:
@nak3 please do not add new template-based tests and use multistage instead, see https://docs.ci.openshift.org/docs/how-tos/migrating-template-jobs-to-multistage/#openshift_installer_src

/hold

Reply:
Ehr, good call :thinking: Why deviate from master and 0.18 here @nak3 ?

Reply:
Thank you. I willl fix it.

> Why deviate from master and 0.18 here @nak3 ?

I generated the file by `make BRANCH=release-v0.19.0 generate-ci-config` in openshift/knative-serving. I will update the script - https://github.com/openshift/knative-serving/blob/master/openshift/ci-operator/generate-ci-config.sh#L27 :sweat_smile:

Reply:
Oh yeah, it's horribly out of date :joy:.

Just copy master and adjust as necessary.